anyatibrian/IRent-backend
# 168926485 sends users account activation link.

#### Description
Users should be able to activate their account on successful registration 
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test
- Hit the endpoint `/api/v1/users/account/:token/activate` that will enable you to activate your account 
- And your account will be successfully activated
Give instructions on how reviewers can verify your work

#### Any background context you want to add
- Account activation enables users to activate their account so that they can be able to login into the system
#### Story-ID

[Starts #168926485]
